ops/web-e2e.cloudbuild.yaml

```diff
@@ -22,10 +35,8 @@ steps:
     args:
       - '-c'
       - |
-        cp -R ../client-libs/ client-libs
-        pip install -r requirements.txt
         python -m flask run --port ${_FLASK_PORT} &
-        cd ../testing/website-e2e && npm install && npm test
+        cd ../testing/website-e2e && npm test
```

Comment:
this is still using a bash entrypoint to run python and npm in one step. we can split further

Reply:
AFAICT these **must** be run in the same step. (`flask` needs to be running _at the same time_ as `npm test`.)

Reply:
ack